model_analyzer/triton/server/server_docker.py

```diff
@@ -93,6 +92,7 @@ def start(self):
             image=self._tritonserver_image,
             device_requests=devices,
             volumes=volumes,
+            volumes_from=[self._host_container],
```

Comment:
Why do we need to mount the volumes from the sibling container?

Reply:
After reading the docs, I think I understood why it is needed. I was wondering can't we figure out the container name so by ourselves when we are inside the container. I came across this post in stackoverflow:

https://stackoverflow.com/questions/20995351/how-can-i-get-docker-linux-container-information-from-within-the-container-itsel

Reply:
Also, we may run the docker mode when the model analyzer is running without a container too. I don't think we can rely on docker volumes for that use case.

Reply:
This just seemed like less of a hack

Reply:
Regarding running outside a container, i put a note about that in the docs, but i will think about a solution to that.  I think remote mode overlaps better in that case.